ios/RNFastCrypto.m

```diff
@@ -180,11 +254,26 @@ + (void) handleDefault:(NSString*) method
         [RNFastCrypto handleDownloadFromClarityAndProcess:method :params :resolve :reject];
     } else if ([method isEqualToString:@"get_transaction_pool_hashes"]) {
         [RNFastCrypto handleGetTransactionPoolHashes:method :params :resolve :reject];
+    } else if ([method isEqualToString:@"allow_processing_task"]) {
+        [RNFastCrypto allowProcessingTasks];
+        resolve(@"{\"success\":true}");
+    } else if ([method isEqualToString:@"stop_processing_task"]) {
+        [RNFastCrypto stopProcessingTasks];
+        resolve(@"{\"success\":true}");
```

Comment:
out of curiosity, why is it not enough to just expose stop_processing_tasks?

Reply:
Do you mean exposing `stopProcessingTasks` function? I intend to reuse the RCT_METHOD map to be able to call it with minimal code changes.

Reply:
Hmm, OK, it's a good idea.

Reply:
It's changed at [eb4746c](https://github.com/ExodusMovement/react-native-fast-crypto/pull/71/commits/eb4746cd08c5d42c429dcdd2ace08c4f9a262683).